exchange/utils.go

```diff
@@ -40,9 +40,20 @@ func splitBidRequest(req *openrtb.BidRequest, impsByBidder map[string][]openrtb.
 	for bidder, imps := range impsByBidder {
 		reqCopy := *req
 		coreBidder := resolveBidder(bidder, aliases)
-		met.AdapterMetrics[coreBidder].RequestMeter.Mark(1)
+		newLabel := pbsmetrics.AdapterLabels{
```

Comment:
This might be a good use-case for embedded structs: https://travix.io/type-embedding-in-go-ba40dd4264df

I've never used these before, so I don't know how well they work in practice... but I _think_ it would let you write this in such a way that `AdapterLabels` picks up all the changes from `Labels` automatically.

Might make this a bit easier to maintain, if you always expect `AdapterLabels` to be a superset of `Labels`.

Not something I'd insist on... or even something that I'm confident would work out better. But I wanted to throw out the possibility in case you hadn't heard of it yet, so you could play around with it.

Reply:
Actually I don't have any use cases in mind to expand the labels. If we want to expand them later, them we can look into type embedding.

Reply:
ok, sounds good.